### ISSUE TITLE
About the results

### ISSUE THREAD

**Issue by wangsu1999** [ikuinen/CMIN#3]
Hi,

I couldn't reach the results of your papers,and the gap was two to three percentage points.Could you please check the codes ?

Thanks,
Looking forward to your reply!

**Comment by ikuinen** [comment]
Hi
I ran the code again on the ActivityCaptions dataset, and still achieved the results even better than the results given by the paper.
The evaluation is performed on the **test** set.
Here is the evaluation results after four epochs of training:
| loss, 0.2260 | mIoU, 0.4372 | IoU@0.1, 0.7938 | IoU@0.3, 0.6310 | IoU@0.5, 0.4308 | IoU@0.7, 0.2476 | IoU@0.9, 0.0964 |

**Comment by wangsu1999** [comment]
Hi 
Thank you !
I tried to increase the learning rate in the model.I used 3e-3 instead of 8e-4, which made me get the results given by the paper.
